tests/cypress/integration/smoke/TransactionSettingsTests.ts

```diff
@@ -0,0 +1,58 @@
+import { TransactionSettings } from '../../../pages/TransactionSettings'
+
+describe('Transactions Settings Smoke Test', () => {
+  before(() => {
+    cy.visit('http://localhost:3000/#/swap?chainId=1')
```

Comment:
We are not always sure on what url we are going to run tests and due to that i think we should use baseUrl value from cypress.json URL. Due to that the url from cy.visit can be simplified to "/#/swap?chainId=1" (rest of the url cypress gonna take from this baseUrl